src/advanced-settings/AdvancedSettings.jsx

```diff
@@ -246,8 +246,9 @@ const AdvancedSettings = ({ intl, courseId }) => {
               </Button>
             ),
             <StatefulButton
+              key="statefulBtn"
               onClick={handleUpdateAdvancedSettingsData}
-              state={isQueryPending && 'pending'}
+              state={isQueryPending ? RequestStatus.PENDING : 'default'}
```

Comment:
Do we really have to use `default` state here not `''`?
Or is there no difference?

Reply:
By default, the `state` prop in the [Paragon library](https://paragon-openedx.netlify.app/components/statefulbutton/#props-api) is set to the string `default`. It seems to me that it is better to follow this rule, although there is no difference)